pkgs/development/ocaml-modules/scfg/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildDunePackage,
+  fetchzip,
+  bos,
+  cmdliner,
+  fmt,
+  fpath,
+  menhir,
+  menhirLib,
+  prelude,
+  sedlex,
+  writableTmpDirAsHomeHook,
+}:
+
+buildDunePackage rec {
+  pname = "scfg";
+  version = "0.5";
+
+  # upstream git repo is misconfigured and cannot be cloned
+  src = fetchzip {
+    url = "https://git.zapashcanon.fr/zapashcanon/scfg/archive/${version}.tar.gz";
+    hash = "sha256-XyNVmI0W0B1JqR+uuojpHe9L5KKLhyoH8vN8+9i7Xcg=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    menhir
+  ];
+
+  buildInputs = [
+    bos
+    cmdliner
+    fmt
+    fpath
+    menhir
+    menhirLib
+    prelude
+    sedlex
+  ];
+
+  checkInputs = [
+    writableTmpDirAsHomeHook
```

Comment:
This is a cool workaround, but really there is an upstream bug (they are not testing what they intend to test) that you may want to report.